microsite/pages/en/index.js

```diff
@@ -373,19 +373,11 @@ class Index extends React.Component {
         </Block>
 
         <ActionBlock className="stripe bg-teal">
-          <ActionBlock.Title>Subscribe to our newsletter</ActionBlock.Title>
-          <ActionBlock.Subtitle>
-            TechDocs is our most used feature at Spotify. Be the first to know
-            when{' '}
-            <a href="https://github.com/spotify/backstage/projects/5">
-              the open source version
-            </a>{' '}
-            ships.
-          </ActionBlock.Subtitle>
+          <ActionBlock.Title>Learn more about TechDocs</ActionBlock.Title>
           <ActionBlock.Link
-            href={`https://mailchi.mp/spotify/backstage-community`}
-          >
-            Subscribe
+            href={`https://backstage.io/docs/features/techdocs/techdocs-overview`}
+    >
+            Docs
```

Comment:
The "Learn more about the service catalog" action is "Read". Should they be the same, or ship this?